src/writeData/containers/ClientLibrariesPage.tsx

```diff
@@ -37,7 +40,22 @@ interface SampleProps {
   sample: string | CodeSampleOption[]
 }
 
-const CodeSampleBlock: FC<SampleProps> = ({name, sample}) => {
+interface ExecuteCodeSampleProps {
+  name: string
+}
+
+export const ExecuteCodeSampleBlock: FC<ExecuteCodeSampleProps> = ({name}) => {
+  const {executeCodeBlock} = useContext(ExecuteCodeBlockContext)
+
+  return (
+    <>
+      <h4>{name}</h4>
+      <CodeSnippet text={executeCodeBlock} />
```

Comment:
This would just be def.execute like the other examples if you use that component to set the variable

Reply:
nope! let's talk :)